### PR TITLE
add a piece of news for KubeCon EU 2022

### PR DESCRIPTION
**Pull request by FeynmanZhou**
Signed-off-by: FeynmanZhou <feynman@kubesphere.io>

### CODE REVIEW

**Comment by FeynmanZhou** [comment]
/assgn @JiannaLiu @Felixnoo 
I write a piece of news for KubeCon EU 2022. Please help to review it, thanks.

File: content/en/news/join-us-at-kubecon-eu-2022.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+[Benjamin Huo](https://kccnceu2022.sched.com/speaker/benjaminhuo) leads the KubeSphere Observability and Serverless team. He is the creator of FluentBit Operator and the founder of the FaaS project OpenFunction (https://github.com/OpenFunction/OpenFunction). He is also the author and architect of several observability open source projects such as Kube-Events, Notification Manager, etc. He loves cloud-native and open source technologies and is the contributor of KEDA, Prometheus Operator, Thanos, Loki, Falco.
+
+**Xiuming Lu - Architect of UISEE**
+
+[Xiuming Lu](https://kccnceu2022.sched.com/speaker/xiuming.lu) is the architect of UISEE who is responsible for system architecture and DevOps of the cloud platform in the autonomous-driving industry. He is experienced in cloud native FaaS and observability areas.
+
+## Session 2: Fluent Operator Intro and Deep Dive
+
+**Please note: this session will hosted as a workshop**
```
Comment (Felixnoo) [inline content/en/news/join-us-at-kubecon-eu-2022.md:40]: `will hosted` -> `will be hosted`

File: content/en/news/join-us-at-kubecon-eu-2022.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+### Abstract
+
+With the support of Fluentd, the original Fluent Bit Operator has been renamed to Fluent Operator. In this session, Fluent Operator maintainers will elaborate on the key features of Fluent Operator as well as its design principles and architecture including:
```
Comment (Felixnoo) [inline content/en/news/join-us-at-kubecon-eu-2022.md:44]: `architecture including` -> `architecture, including`

File: content/en/news/join-us-at-kubecon-eu-2022.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+1. From Fluent Bit Operator to Fluent Operator.
+
+2. Fluent Operator design principles.
+
+3. Fluent Operator Architecture.
+
+4. Use Fluent Bit as a light-weighed logging agent on Kubernetes.
```
Comment (Felixnoo) [inline content/en/news/join-us-at-kubecon-eu-2022.md:52]: `light-weighed` -> `light-weight`

File: content/en/news/join-us-at-kubecon-eu-2022.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+
+8. Community & Roadmap.
+
+### Speakers
+
+**Benjamin Huo - Senior Architect at QingCloud** 
+
+Benjamin Huo led the KubeSphere Observability and Serverless team. He is the creator of FluentBit Operator and the founder of the FaaS project [OpenFunction](https://github.com/OpenFunction/OpenFunction), also the author and architect of several observability open source projects such as Kube-Events, Notification Manager, etc. He loves cloud-native and open source technologies and is the contributor of Prometheus.
+
+**Senior Software Engineer at QingCloud**
```
Comment (Felixnoo) [inline content/en/news/join-us-at-kubecon-eu-2022.md:68]: Missing a name here?

File: content/en/news/join-us-at-kubecon-eu-2022.md [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+
+Han is Fluent operator maintainer, KubeSphere/Fluent/OpenFunction member, and he is also KubeEdge contributor. Han is interested in open source and cloud-native technologies in devops/observability/serverless/edge computing areas.
+
+## Session 3: Build a Cloud Native Logging Pipeline on the Edge with Fluent Operator
+
+### Abstract
+
+Fluent Operator formerly known as Fluentbit Operator, provides great flexibility in building a logging layer based on Fluent Bit and Fluentd. It was created by the KubeSphere community to solve the following problems: 
+
+1. Collect K8s logs through a light-weighted agent like Fluent Bit
```
Comment (Felixnoo) [inline content/en/news/join-us-at-kubecon-eu-2022.md:78]: `light-weighted` -> `light-weight`

**Review by Felixnoo — COMMENTED** [review]
Please see my comments, thanks.

File: content/en/news/join-us-at-kubecon-eu-2022.md [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+
+**Senior Software Engineer at QingCloud**
+
+Han is Fluent operator maintainer, KubeSphere/Fluent/OpenFunction member, and he is also KubeEdge contributor. Han is interested in open source and cloud-native technologies in devops/observability/serverless/edge computing areas.
+
+## Session 3: Build a Cloud Native Logging Pipeline on the Edge with Fluent Operator
+
+### Abstract
+
+Fluent Operator formerly known as Fluentbit Operator, provides great flexibility in building a logging layer based on Fluent Bit and Fluentd. It was created by the KubeSphere community to solve the following problems:
```
Comment (JiannaLiu) [inline content/en/news/join-us-at-kubecon-eu-2022.md:76]: Fluent Operator formerly known as Fluentbit Operator, -> Fluent Operator, formerly known as Fluentbit Operator,

**Comment by FeynmanZhou** [comment]
@Felixnoo @JiannaLiu I have updated according to your comments. Thanks!

**Pull request merged**